floss/language/rust/decode_utf8.py

```diff
@@ -0,0 +1,112 @@
+# Copyright (C) 2023 Mandiant, Inc. All Rights Reserved.
+import sys
+import logging
+import pathlib
+import argparse
+from typing import List, Tuple, Iterable, Optional
+
+import pefile
+
+MIN_STR_LEN = 4
+
+logger = logging.getLogger(__name__)
+
+
+def get_rdata_section(pe: pefile.PE) -> pefile.SectionStructure:
+    for section in pe.sections:
+        if section.Name.startswith(b".rdata\x00"):
+            return section
+
+    raise ValueError("no .rdata section found")
+
+
+def extract_utf8_strings(pe: pefile.PE, min_length=MIN_STR_LEN) -> List[Tuple[str, int, int]]:
```

Comment:
can you add a function that's signature is analog to `extract_ascii_strings`, i.e. operate on buffers?
this can then be wrapped by this or a similar function.